### PR TITLE
Added chkconfig compatibility and license to init.d script

### PR DESCRIPTION
**Pull request by vgarciag**
This PR resolves issue https://github.com/telefonicaid/IoT-STH/issues/129

### CODE REVIEW

File: package/rpm/SOURCES/etc/init.d/sth [hunk 1]
```diff
@@ -1,5 +1,36 @@
 #!/usr/bin/env bash
 
+# Copyright 2015 Telefonica Investigación y Desarrollo, S.A.U
+#
+# This file is part of Short Term Historic (FI-WARE project).
```
Comment (gtorodelvalle) [inline package/rpm/SOURCES/etc/init.d/sth]: I think we currently are "FIWARE" instead of "FI-WARE", right? :)

Comment (frbattid) [inline package/rpm/SOURCES/etc/init.d/sth]: It would not be good Juanjo sees the hyphen xD

Comment (vgarciag) [inline package/rpm/SOURCES/etc/init.d/sth]: :worried: Ops. Fixed!

**Comment by gtorodelvalle** [comment]
LGTM! Thanks @vgarciag ;) Landing and removing the branch... ;)

**Pull request merged**